### ISSUE TITLE
Mute / con sord. sounds like pizzicato on bowed strings

### ISSUE THREAD

**Issue by Dima-S-Jr** [musescore/MuseScore#22336]
### Issue type

General playback bug

### Steps to reproduce

1. enter any notes/group of notes
2. add the text instruction _mute_ (translated into my language, this instruction is called "_con sord._") to any of the entered notes.
3. activate the _MuseScore Basic_ profile
4. click play

### Screenshots/Screen recordings

https://github.com/musescore/MuseScore/assets/134041121/71062f0e-a980-4955-88f4-d4744399c306

### MuseScore Version

OS: Windows 10 Version 2009 MuseScore version (64-bit): 4.2.1-240230937, revision: github-musescore-musescore-d757433

### Regression

I don't know

### Operating system

Windows 11

### Additional context

This only occurs with the _MuseScore Basic_ profile.

**Comment by wizofaus** [comment]
Just found this myself - seems like it should be an easy sort of thing to fix, but already being worked on?

**Comment by cbjeukendrup** [comment]
@wizofaus It doesn't look like it, so feel free to give this a try!

**Comment by MarcSabatella** [comment]
Since there presumably aren’t mute samples in MS Basic (or other GM soundfonts), I guess the proposed fix is to simply ignore the direction?  Or perhaps somehow diminish the volume?

**Comment by wizofaus** [comment]
Whatever the normal behaviour is when no sample exists - I would assume to ignore it. Of course you can have mute and pizzicato both active with string instruments, so I don't how the system would handle that were we ever to have more complete samples.
You can already adjust the dynamic level by other means (and I'd even suggest that if a section is marked "mf con sord" it should still played at a dynamic level essentially the same as "mf senza sord").

**Comment by DaddyLudwig** [comment]
Yeah con sord isn't really a volume thing, it's a timbre thing.

**Comment by wizofaus** [comment]
Anyway for now I've just modified the "articulation" maps so that "mute" maps to a sensible MIDI program.
I've also changed it so that the "mute" technique only changes *brass* instruments to use the MIDI "muted trumpet" preset (previously it applied to everything single wind instrument).  Technically I fudged this by just checking that the enum value was greater than that for "Horn", which is the lowest-valued MIDI brass instrument preset. Obviously a long way from ideal, but a great deal better than having, e.g. muted clarinets sound like trumpets (accepted, muted clarinet isn't often called for).
